client/src/components/addRecipe.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+
+
+class addRecipe extends Component {
+  //if the user is editing a current recipe, pass the name and intructions as props
```

Comment:
Just a suggestion; maybe adding some more detail to these comments would help make your code more intuitive for less experienced programmers